src/transformState.js

```diff
@@ -6,6 +6,26 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, ...[action.extraData]);
+        break;
+
+      case 'removeProperties':
+        for (const j of action.keysToRemove) {
+          delete state[j];
+        }
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+    }
```

Comment:
Follow the checklist, please. Add a default case
<img width="789" alt="image" src="https://user-images.githubusercontent.com/99403198/233780675-fb124773-9ec4-427b-9a8a-f9e3f0e22183.png">